code/solutions/linting-and-testing/example_sqs_queue.yaml

```diff
@@ -0,0 +1,23 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - linting and testing - example SQS queue (uksb-1q9p31idr)
+
+Resources:
+  MyQueue:
```

Comment:
Please rename `MyQueue` to something more descriptive e.g. `CfnWorkshopQueue` or just plain simple `SqsQueue`

Reply:
Thank you, @rezabekf !  I will use `SqsQueue` for consistency with your suggestions on relevant names in the `Outputs` section.